cli/cmd/token/issue.go

```diff
@@ -21,10 +21,17 @@ func IssueCmd(ch *cmdutil.Helper) *cobra.Command {
 				return err
 			}
 
+			if displayName == "" {
```

Comment:
Respect the `--interactive=false` flag:

```suggestion
			if ch.Interactive && displayName == "" {
```